### PR TITLE
adds fabricator setup for item & category

### PR DESCRIPTION
**Pull request by meyerhoferc**
adds fabricator to gemfile
adds fabricator directory to spec with fabricator for item & category
* still needs category relationship for an item
* still needs image for an item

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -42,6 +42,7 @@ group :development, :test do
   gem 'launchy'
   gem 'shoulda-matchers', '~> 3.0'
   gem 'fabrication'
+  gem 'faker'
```
Comment (CjMoore) [inline Gemfile]: I pulled it down and played around with it. I kept getting an uninitialized constant Faker::HarryPotter error. I found a solution. It has to look this in the gem file:  gem 'faker', github: 'stympy/faker.git'

File: Gemfile [hunk 2]
```diff
@@ -42,6 +42,7 @@ group :development, :test do
   gem 'launchy'
   gem 'shoulda-matchers', '~> 3.0'
   gem 'fabrication'
+  gem 'faker', github: 'stympy/faker.git'
```
Comment (cmacaulay) [inline Gemfile:45]: @meyerhoferc @CjMoore I made and committed this change, should ensure that the faker gem works correctly.

File: spec/fabricators/fabricator.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+require 'faker'
```
[the remark this answers is not in the document]
Comment (cmacaulay) [reply spec/fabricators/fabricator.rb:1]: 👎

**Review by cmacaulay — APPROVED** [review]
Looks great @meyerhoferc! @valerietrudell @CjMoore I am going to merge these changes.

**Pull request merged**